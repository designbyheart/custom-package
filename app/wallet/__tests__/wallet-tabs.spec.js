import React from 'react'
import renderer from 'react-test-renderer'
// import { walletRoute } from '../../common'
// import { getNavigation } from '../../../__mocks__/static-data'
// import WalletTabs from '../wallet-tabs'

// function props() {
//   return {
//     navigation: getNavigation(),
//     route: {
//       currentScreen: walletRoute,
//     },
//   }
// }

describe('<WalletTabs />', () => {
  xit('should render WalletTabs', () => {
    // const wrapper = renderer.create(<WalletTabs />).toJSON()
    // expect(wrapper).toMatchSnapshot()
  })
})
