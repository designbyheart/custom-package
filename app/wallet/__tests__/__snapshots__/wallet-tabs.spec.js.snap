// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WalletTabs /> should render properly and match the snapshot 1`] = `
<View
  screenProps={
    Object {
      "navigation": Object {
        "dispatch": [MockFunction],
        "getParam": [MockFunction],
        "goBack": [MockFunction],
        "isFocused": [MockFunction],
        "navigate": [MockFunction],
        "push": [MockFunction],
        "setParams": [MockFunction],
        "state": Object {
          "params": Object {},
        },
      },
    }
  }
/>
`;
